thinkhazard/security.py

```diff
@@ -0,0 +1,12 @@
+import os
+from passlib.apache import HtpasswdFile
+
+
+def groupfinder(username, password, request):
+    ht = HtpasswdFile()
+    ht.load_string(os.environ.get("HTPASSWORDS", ""))
```

Comment:
passlib doc is unclear to me, what format takes this method? could be pass multiple users?

Reply:
This method take unicode or bytes, like `f.read()`.
Yes could pass multiple passwords, env var contains the htpassword file content.